run.sh

```diff
@@ -119,7 +164,16 @@ while true; do
 
   # Ping kubelet for liveness
   f=kubelet_ping
-  c=$(curl -m 5 http://127.0.0.1:10255/healthz 2>&1 || true)
+
+  set +e
+  if ! c=$(curl -m "5" -f -s -S http://127.0.0.1:10248/healthz 2>&1); then
```

Comment:
is the health port 10255 or 10248?

Reply:
10248 is default healthz port, see https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/ under "healthz-port". This is same as existing health-monitor.sh script -- https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/health-monitor.sh#L75